### PR TITLE
Upgrade librdkafka to v1.3.0

### PR DESCRIPTION
**Pull request by oandrew**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Upgrade librdkafka to v1.3.0
Allow using internal  librdkafka on Mac
Allow using internal  libgsasl on Mac (rdkafka dependency)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -64,7 +64,7 @@
 	url = https://github.com/GNOME/libxml2.git
 [submodule "contrib/libgsasl"]
 	path = contrib/libgsasl
-	url = https://github.com/ClickHouse-Extras/libgsasl.git
+	url = https://github.com/oandrew/libgsasl.git
```
Comment (oandrew) [inline .gitmodules]: make sure to change back once https://github.com/ClickHouse-Extras/libgsasl/pull/2 is merged

**Comment by oandrew** [comment]
Can someone restart the build? Looks like it's stuck

**Comment by oandrew** [comment]
@filimonov

**Comment by filimonov** [comment]
LGTM

**Review by filimonov — APPROVED** [review]

**Comment by oandrew** [comment]
@alexey-milovidov

**Pull request merged**